src/less/day-slot.less

```diff
@@ -30,20 +31,25 @@
     min-height: 1em;
   }
 
+  .rbc-time-slot {
+    border-top: 1px solid lighten(@cell-border, 10%);
+  }
 }
 
 .rbc-time-slot {
   flex: 1 0 0;
-  min-height: 20px;
-  height: 20px;
 
-  & + & {
-    border-top: 1px solid @cell-border;
+  &.rbc-now {
+    font-weight: bold;
   }
+}
 
-  &:nth-child(even){
-    border-top-color: lighten(@cell-border, 10%);
-  }
+.rbc-timeslice-group {
+  border-bottom: 1px solid @cell-border;
+  min-height: 40px;
+  height: 40px;
```

Comment:
Gotcha. So we use simple flex for the time slice groups and let the Calendar box size them, with a min-height of 40 and no max height

via CloudMagic Email [https://cloudmagic.com/k/d/mailapp?ct=pa&cv=8.5.38&pv=5.1.1&source=email_footer_2] On Sun, Jun 05, 2016 at 1:48 PM, Jason Quense < notifications@github.com [notifications@github.com] > wrote:
In src/less/day-slot.less [https://github.com/intljusticemission/react-big-calendar/pull/107#discussion_r65822319] :

>  
> -  &:nth-child(even){
> -    border-top-color: lighten(@cell-border, 10%);
> -  }
> +.rbc-timeslice-group {
> +  border-bottom: 1px solid @cell-border;
> +  min-height: 40px;
> +  height: 40px;


I think we are on the same page, I would just say that at a minimum the columns should extend the heightof the calendar so we don't have have a mostly empty bordered box, Whether or not the columns "grow" the calendar itself or overflow with a scrollbar is up to the user and whether they apply min-heights on the slices/groups. From a default perspective tho I think the tweaks in my branch (minus the removal of the min-height) should be reasonable

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub [https://github.com/intljusticemission/react-big-calendar/pull/107/files/9718a53bca39efd97a14729ebb7a92bd3ffbb4f2#r65822319] , or mute the thread [https://github.com/notifications/unsubscribe/AAF_yvNssdMTfPJNFs8_YulAZqZogASYks5qIxoCgaJpZM4IuR6m] .